i18n/en.json

```diff
@@ -2963,6 +2963,8 @@
   "navbar.toggle2": "Toggle sidebar",
   "navbar.viewInfo": "View Info",
   "navbar.viewPinnedPosts": "View Pinned Posts",
+  "no_results.channels.subtitle": "Check the spelling or try another search.",
+  "no_results.channels.title": "No results for {channelName}",
```

Comment:
Recommend using a generic name for the both keys here:
```
  "no_results.search.subtitle": "Check the spelling or try another search.",
  "no_results.search.title": "No results for {searchText}",
```
Or maybe change the variant name to `ChannelSearch`